public/docs/_examples/i18n/e2e-spec.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path='../_protractor/e2e.d.ts' />
+'use strict';
```

Comment:
We are using it in all examples, I think it has to do with ts-node to be able to understand a few things.